macros/blueprint-proc-macro-playground/src/lib.rs

```diff
@@ -104,6 +104,17 @@ fn report_service_health(uptime: f64, response_time: u64, error_rate: f64) -> Ve
     issues.concat()
 }
 
+// ==================
+//   Benchmarks
+// ==================
+#[benchmark(job_id = 0, threads = 2)]
+fn keygen_2_of_3() {
```

Comment:
The job was parameterized by (n,t), I think we should somehow extend the `params(n,t)` in a way that allows you to specify ranges for numbered parameters, thoughts?

Reply:
Yes we can do that, however the mechanism for doing this automatically is a bit complex (i.e matching the right t and n). A better approach would be a function generator that takes `i` as the run number and output the parameters for the job based on `i`.